go/consensus/api/grpc.go

```diff
@@ -184,11 +190,11 @@ var (
 )
 
 func handlerSubmitTx(
-	srv interface{},
+	srv any,
```

Comment:
Should we do a commit that updates these across all grpc backends?

Reply:
Actually, I can replace it everywhere, except in `aesm_proto.pb.go`.